site/docs/streams/streams.md

```diff
@@ -80,15 +80,19 @@ def randomMessage: Stream[IO, XAddMessage[String, String]] = Stream.eval {
   }
 }
 
-for {
-  client    <- Stream.resource(RedisClient[IO].from("redis://localhost"))
-  streaming <- RedisStream.mkStreamingConnection[IO, String, String](client, stringCodec)
-  source    = streaming.read(Set(streamKey1, streamKey2), chunkSize = 1)
-  appender  = streaming.append
-  _ <- Stream(
-         source.evalMap(putStrLn(_)),
-         Stream.awakeEvery[IO](3.seconds) >> randomMessage.through(appender)
-       ).parJoin(2).void
-} yield ()
+def runStream(): Stream[IO, Unit] = {
```

Comment:
I'd go with an object `StreamingDemo extends IOApp.Simple` instead. 

And just to share my opinion, I don't think we should ever define methods with empty parens, that's a Javaism which is completely unnecessary when working with purely functional programming 😉 

I know a lot of other libraries doing that and I can't even look at it 😄

Reply:
Okay thanks for the tips! I'll try to keep that in mind - still trying to kick away all my Python habits 😄 

I'll do the rewrite later when I have some time.